macos/mac_os_resource_snd.ksy

```diff
@@ -75,12 +75,8 @@ types:
         doc: number of samples in array
       - id: sample_rate
         -orig-id: sampleRate
-        type: u2
-        doc: sample rate (Fixed Point, Integer part)
-      - id: sample_rate_frac
-        -orig-id: sampleRate
-        type: u2
-        doc: sample rate (Fixed Point, Fraction part)
+        type: unsigned_fixed_point
+        doc: sample rate
```

Comment:
The `doc` key should not be included if it's only a trivial transcription of the `id`, see [KSY Style Guide > Sequence attributes](https://doc.kaitai.io/ksy_style_guide.html#seq-attr):

> Every key is optional. Attributes SHOULD have at least `id` and `doc` — however, see below for notes about omitting `id`, and `doc` SHOULD NOT be included if it’s trivial (i.e. if it is a copy of `id`, and there is really nothing more to say about that attribute).

```suggestion
```